staging/src/k8s.io/apiserver/pkg/endpoints/handlers/fieldmanager/fieldmanager.go

```diff
@@ -233,8 +233,12 @@ var stripSet = fieldpath.NewSet(
 	fieldpath.MakePathOrDie("metadata", "name"),
 	fieldpath.MakePathOrDie("metadata", "namespace"),
 	fieldpath.MakePathOrDie("metadata", "creationTimestamp"),
+	fieldpath.MakePathOrDie("metadata", "deletionTimestamp"),
 	fieldpath.MakePathOrDie("metadata", "selfLink"),
 	fieldpath.MakePathOrDie("metadata", "uid"),
+	fieldpath.MakePathOrDie("metadata", "clusterName"),
+	fieldpath.MakePathOrDie("metadata", "generation"),
+	fieldpath.MakePathOrDie("metadata", "managedFields"),
```

Comment:
We need a test for this method, we don't know if it works for nested fields like this one?

Reply:
Do you mean a test that the fields get stripped?
Or that MakePathOrDie works?

About the managedFields in there: @lavalamp you meant that we should rather strip the field here and that I should remove the other place where they get removed.
I think the current way they get removed is better than stripping here afterwards. Also because stripping it later means the fields will get merged which has a cost and complexity.

Reply:
> Do you mean a test that the fields get stripped?

This :-)

`MakePathOrDie` lives somewhere else and should (is?) be tested there.